src/pybind/mgr/volumes/fs/nfs.py

```diff
@@ -195,9 +317,21 @@ def __init__(self, mgr, namespace=None):
         self.mgr = mgr
         self.rados_pool = 'nfs-ganesha'
         self.rados_namespace = namespace #TODO check if cluster exists
-        self.export_conf_blocks = []
         self.exports = {}
 
+        try:
+            log.info("Begin export parsing")
+            for cluster_id in available_clusters(self.mgr):
+                cluster_id = cluster_id[8:]
```

Comment:
I think [8:] is not intuitive. Maybe cluster_id[len(CLUSTER_PREFIX):], where CLUSTER_PREFIX = 'ganesha-'  is at the module level or class attribute of NFSCluster

Reply:
I have added a comment for it.